gpt_commit_msg.py

```diff
@@ -114,9 +111,17 @@ def main():
     else:
         args.model = "gpt-3.5-turbo"
 
+    if args.prompt:
+        prompt = args.prompt
+    else:
+        prompt = """Write a git commit message for the following. The message
+                starts with a one-line summary of 60 characters, followed by a
+                blank line, followed by a longer but concise description of the
+                change."""
+
```

Comment:
Instead of doing this, can you pass the default prompt using `default=` as a parameter to `--prompt` argument?